### PR TITLE
Add pregenerated certificates to the repo and use them by default.

### PR DESCRIPTION
**Pull request by jgraham**
In many situations it's inconvenient to have to install OpenSSL to
generate certificates; for example on Windows. In mozilla-central we
solve this by checking in some pregenerated certificates to the
repository. We can use the same solution (and the same certificates)
here.

This also changes |wpt run| and |wpt serve| to use those certificates
by default.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/6938)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2017-08-18 17:44:31
Finished: 2017-08-18 18:03:38



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17563)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266060650)

File: README.md [hunk 1]
```diff
@@ -187,9 +177,31 @@ then remove the `tools` and `resources` directories, as above.
...
+Alternatively, you may also use
+[Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about)
+in the Windows 10 Anniversary Update build, then access your windows
+partition from there to launch `wpt` commands.
+
+Certificates
+============
+
+By default pregenerated certificates for the web-platform.test domain
+are provided in the repopsitory. If you wish to generate new
```
Comment (bobholt) [inline README.md]: repository (spelling)

File: README.md [hunk 2]
```diff
@@ -187,9 +177,31 @@ then remove the `tools` and `resources` directories, as above.
...
+certificates for any reason it's possible to use OpenSSL when starting
+the server, or starting a test run, by providing the
+`--ssl-type=openssl` argument to the `wpt serve` or `wpt run`
+commands.
+
+If installed OpenSSL in such a way that running `openssl` at a
```
Comment (bobholt) [inline README.md]: If _you_ installed...

**Review by bobholt — APPROVED** [review]
Approved with nits and assuming tests can be made to pass.

**Pull request merged**